.gitignore

```diff
@@ -4,3 +4,19 @@ builder/envs/**
 *.log
 templater/results/**
 
+# Maven Output #
+#################
+target
+
+# Intellij IDEA Project Files
```

Comment:
shouldn't this be in a global git ignore file on a devs machine? if not - we need to add the git ignores for emacs, eclipse, sublime and so on

Reply:
I just copied these lines from the main project, assuming that if it's ok for the main project, then it's ok for perf-harness too. Don't know ignores for other editors, so decided to copy-paste section "# Actual code editors" from the main project.